### ISSUE TITLE
[BUG] fltk 1.3.22 fails to build documentation

### ISSUE THREAD

**Issue by utopiabound** [fltk-rs/fltk-rs#1322]
https://docs.rs/crate/fltk/1.3.22/builds

## Describe the bug
docs.rs is missing documentation for 1.3.22, as it fails to build.

**Comment by MoAlyousef** [comment]
Yeah I noticed this right after publishing 1.3.22. The problem is that cargo changed the arguments for -Zrustdoc-scrape-examples to not take an argument in the latest Cargo:
https://github.com/rust-lang/cargo/pull/10343

I've updated the code in the Cargo.toml after seeing the docs build failing on docs.rs, and am contemplating releasing a new version just to get the docs to build.

**Comment by MoAlyousef** [comment]
This should be fixed in the latest release (1.3.23)

https://docs.rs/fltk/latest/fltk/
